mithril-aggregator/src/event_store/runner.rs

```diff
@@ -2,38 +2,39 @@ use anyhow::Context;
 use mithril_common::logging::LoggerExtensions;
 use mithril_common::StdResult;
 use slog::{debug, info, Logger};
-use sqlite::Connection;
-use std::{path::PathBuf, sync::Arc};
+use sqlite::ConnectionThreadSafe;
```

Comment:
You should use our aliased type: `SqliteConnection`

Reply:
Done